code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -280,8 +280,6 @@
 	glass_icon_state = "glass_brown"
 	glass_name = "glass of coffee"
 	glass_desc = "Don't drop it, or you'll send scalding liquid and glass shards everywhere."
-	effective_blood_max = 240
```

Comment:
This and other changes like this would mess with the blood system, in that this stuff wouldnt be as effective for blood replacement.

Reply:
do you want me to remove that specific line?

Reply:
nevermind, I'm blind, one second and it'll be put up!